test/compilable/objc_interface_final_19654.d

```diff
@@ -0,0 +1,7 @@
+// EXTRA_OBJC_SOURCES
```

Comment:
Q: I looked at `d_do_test` can it looks this doesn't have any real effect when empty?

Reply:
The effect is that it should only be run on platforms which supports Objective-C.